diagrams.md

```diff
@@ -0,0 +1,548 @@
+        Currency funds_obligated
+        Choice incrementally_funded
+    }
+    CLIN {
+        GUID sys_id PK
+        Reference task_order FK
+        String clin_number
+        Choice clin_status
+        Choice idiq_clin
+        Date pop_start_date
+        Date pop_end_date
+        Currency funds_total
+        Currency funds_obligated
+    }
+    CLOUD-SERVICE-PROVIDER {    
+        GUID sys_id PK
+        String name
+        Choice network
+        String uri "change to URL type?"
+        URL pricing_calculator_uri
+    }
+    COSTS {
+        GUID sys_id PK
+        Reference csp FK
+        Reference portfolio FK
+        Reference organization FK
+        Reference agency FK
+        String task_order_number
+        String clin
+        Boolean is_actual "actual/forecast"
+        Currency value
+        Date year_month "ignore day"
+    }
+    AGENCY {
+        GUID sys_id PK
+        Integer css_id "Contract Support System"
+        String label
+        String title
+        String acronym
+    }
+    OPERATOR {
+        GUID sys_id PK
+        Reference portfolio FK
+        String dod_id
+        String email
+        Boolean needs_reset
+        Boolean provisioned
+        DateTime provisioning_request_date
+        DateTime provisioned_date
+        String provisioning_failure_cause
+        String added_by
+    }
+    PORTFOLIO ||--|{ TASK-ORDER : "funded by"
+    TASK-ORDER ||--|{ CLIN : "has multiple (from EDA)"
+    CLIN }|--|| TASK-ORDER : "part of"
+    PORTFOLIO ||--|{ COSTS : "has (from CSPs)"
+    PORTFOLIO ||--|{ SYS_USER : "portfolio managers and viewers"
+    PORTFOLIO ||--|{ OPERATOR : "pending operators"
+    COSTS }|--|| CLOUD-SERVICE-PROVIDER : "billed by"
+    COSTS }|--|{ TASK-ORDER : "billed against"
+    COSTS }|--|| ORGANIZATION : "incurred by"
+    COSTS }|--|| AGENCY : ""
+    ALERT }|--|| PORTFOLIO : ""
+    ALERT }|--|| TASK-ORDER : ""
+    ALERT }|--|| CLIN : "relates to"
+
+    %% Package
+    DITCO-CONTRACT-SPECIALIST {
+        GUID sys_id PK
+        Reference user FK "to sys_user"
+        String first_name
+        String last_name
+        Boolean is_ko
+        Integer css_id "Contract Support System"
+    }
+    IDIQ-CLIN {
+        GUID sys_id PK
+        Reference classification_level FK
+        Choice contract "JWCC"
+        Choice contract_type "FFP, T&M"
+        String description
+        String idiq_clin
+    }
+    ACQUISITION-PACKAGE {
+        GUID sys_id PK
+        Reference classification_level FK
+        Reference contract_award FK
+        Reference contract_considerations FK
+        List contract_modifications FK
+        Reference contract_type FK
+        List contributors FK "to sys_user"
+        List mission_owners FK "to sys_user"
+        Reference current_contract_information FK
+        Reference current_environment FK
+        Reference ditco_ko FK
+        Reference fair_opportunity FK
+        List funding_plans FK
+        Reference funding_request FK
```

Comment:
Is the `funding_request` on the package different from the one in `Funding Requirement.funding_request`? They appear like duplicates since the addition of the `Funding Requirement` table, so I'm just wondering.

Reply:
It's the same funding request.  Would like to remove this column/relationship and refactor via route through Funding Requirement.